README.md

```diff
@@ -1,3 +1,25 @@
 # magpi_scraper
 
 Downloads all issues of MagPi from https://magpi.raspberrypi.org/
+
+## Requipments
+
+* Python 3
+
+## Setup
+
+You will need to install the following with `pip3` (or `pip` depending
+on your environment):
+
+```bash
+pip3 install requests
+pip3 install bs4
+```
+
+## Usage
+
+From the project directory:
+
+```bash
+python3 magpi_scraper.py
+```
```

Comment:
Just some boilerplate instructions that I needed to do to get this running on my computer.